client/src/components/statistics/SeriesChart.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, Label } from 'recharts';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft, faArrowRight } from '@fortawesome/free-solid-svg-icons';
+
+class SeriesChart extends React.Component {
+  state = {
```

Comment:
After removing `React.Component` and all that, use `useState` hook and then `setPage`, `setPageSize`